ipaserver/install/server/install.py

```diff
@@ -940,9 +940,22 @@ def install(installer):
 
     print("======================================="
           "=======================================")
-    print("Setup complete")
+    print("IPA server installation successful")
     print("")
-    print("Next steps:")
+    print("You can access Web UI using https://%s/ipa/ui" % (host_name))
+    print("\nObtain a Kerberos Ticket for issuing IPA command 'kinit admin'.")
+    print("\nServer is installed with following roles :")
```

Comment:
In English, there is no space between before a colon. There are three other cases of ``text :`` below.